.circleci/config.yml

```diff
@@ -96,17 +105,25 @@ jobs:
           name: "Check current java version"
           command: java -version
       - run:
-          name: "Install java 15"
+          name: "Install java 17"
           command: |
-            brew tap adoptopenjdk/openjdk
-            brew install adoptopenjdk15
```

Comment:
This project is now dead. Java 17+ not supported
New import requires a symlink

Reply:
Nice workaround